### PR TITLE
Update featured-dotgov.html

### PR DESCRIPTION
**Pull request by amcelya**
Update featured .gov to everykidoutdoors.gov

## Changes

- Updates featured .gov from Read.gov to EveryKidOutdoors.gov
- Updates description to match new featured domain

## Context for reviewers

Please review the language and grammar to ensure it is functional, succinct, and correct. The original language can be found here:
[https://docs.google.com/document/d/1jkiWrB1YCdhTD85xWDzJhg3gb0Sk_HnXYg-2SjGeDko/edit]

### CODE REVIEW

File: _includes/featured-dotgov.html [hunk 1]
```diff
@@ -6,9 +6,9 @@ <h2 class="usa-alert__heading font-sans-xl">
         Featured .gov
       </h2>
       <h4 class="font-heading-lg text-semibold margin-top-205 margin-bottom-205">
-        <a class="usa-link usa-link--external" rel="noopener noreferrer" href="https://read.gov/">Read.gov</a>
+        <a class="usa-link usa-link--external" rel="noopener noreferrer" href="https://everykidoutdoors.gov/index.htm">EveryKidOutdoors.gov</a>
```
Comment (h-m-f-t) [inline _includes/featured-dotgov.html:9]: https://everykidoutdoors.gov does redirect to the /index.htm page, but I think you should drop `/index.htm` from this href.

**Review by h-m-f-t — CHANGES_REQUESTED** [review]
Suggesting a small change, but everything else looks good!

**Review by amcelya — COMMENTED** [review]
Changes made as requested

**Review by amcelya — COMMENTED** [review]
Changes made

**Review by h-m-f-t — APPROVED** [review]

**Pull request merged**